client/hotrod-client/src/test/java/org/infinispan/client/hotrod/hash/ConsistentHashTest.java

```diff
@@ -46,9 +41,10 @@ public void testKeysMapToCorrectSegment() {
 
       for (int i = 0; i < 100; i++) {
          Object keyValue = kv(i);
-         ((ControlledChannelFactory) rcm.getChannelFactory()).useOnFetch((server, op) -> {
-            requests.put(keyValue, server);
-         });
+         // TODO: change this to something new
```

Comment:
Seems this test isn't doing anything.

Reply:
:+1: Let me fix this up, missed it.

Reply:
Talking with @jabolina he noted we have https://issues.redhat.com/browse/ISPN-8098 which should be the way this is implemented.